curriculum/challenges/english/25-front-end-development/quiz-css-positioning/66ed8ffcf45ce3ece4053eb5.md

```diff
@@ -17,438 +17,594 @@ To pass the quiz, you must correctly answer at least 17 of the 20 questions belo
 
 #### --distractors--
 
-When combined with `position: absolute` or `position: relative`.
+```css
+.clearfix::before {
+  position: fixed; 
+  top: 0; 
+  width: 100%; 
+  clear: both;
+}
+```
 
 ---
 
-When its parent element has `z-index: auto` and `position: fixed`.
+```css
+.clearfix::after {
+  position: relative;
+  top: 30px;
+  left: 30px;
+  clear: all;
+}
+```
 
 ---
 
-When it is combined with `float: left`, if two sibling elements are of the same `z-index` and absolutely positioned.
+```css
+.clearfix::before {
+  top: 30px;
+  clear: none;
+}
+```
 
 #### --answer--
 
-In combination with `position: fixed` or `position: sticky`.
+```css
+.clearfix::after {
+  content: "";
+  display: block;
+  clear: both;
+}
+```
 
 ### --question--
 
 #### --text--
 
-If two sibling elements have the same `z-index` and are both positioned absolutely using `position: absolute`, what determines which one is on top?
+What is static positioning?
 
 #### --distractors--
 
-It rearranges the sibling elements in relation to its new position.
+This is used to remove an element from its normal flow on the page and automatically position it in the upper right hand side of the webpage.
 
 ---
 
-Adding `z-index: auto` to the parent container.
+This allows you to take an element out of the normal document flow, making it behave independently from other elements.
 
 ---
 
-Using `position: relative` on the parent container.
+This allows an element to stick to a defined position only when you scroll past a certain point.
 
 #### --answer--
 
-The document source order (HTML position).
+This is the normal flow for the document. Elements are positioned from top to bottom and left to right one after another.
 
 ### --question--
 
 #### --text--
 
-In what way will the property `position: relative` exert an effect on sibling elements in regards to document flow?
+Which of the following is an example of setting the navbar to the top of the page using fixed positioning?
 
 #### --distractors--
 
-It removes the element from the normal document flow, which affects the positions of siblings.
+```css
+.navbar {
+  fixed: top; 
+  top: 0; 
+  width: 100%; 
+}
+```
 
 ---
 
-The relative element will move by the parent's top and left properties. 
+```css
+.navbar {
+  upper: fixed; 
+  width: 100%; 
+}
+```
 
 ---
 
-It creates a new stacking context which lifts the element in relation to any siblings.
+```css
+.navbar {
+  position: fixed-top; 
+  top: 0; 
+  width: 100%; 
+}
+```
 
 #### --answer--
 
-It allows the element to shift without affecting the document flow of sibling elements.
+```css
+.navbar {
+  position: fixed; 
+  top: 0; 
+  width: 100%; 
+}
+```
 
 ### --question--
 
 #### --text--
 
-What happens when a `position: relative` element is nested inside of a `position: absolute` parent, with `top` and `left` values set?
+Which of the following is a valid value to use for the `z-index` property?
 
```

Comment:
I think the answers to this question should include values only instead of `z-index: value;`